django_s3/utils.py

```diff
@@ -59,7 +59,7 @@ def is_metadata_xml_file(file_path):
     """
     from hs_file_types.enums import AggregationMetaFilePath
 
-    if not (file_path.endswith(AggregationMetaFilePath.METADATA_FILE_ENDSWITH)
-            or file_path.endswith(AggregationMetaFilePath.RESMAP_FILE_ENDSWITH)):
+    if not (file_path.endswith(AggregationMetaFilePath.METADATA_FILE_ENDSWITH.value)
```

Comment:
Is this necessary? I just pulled the python:3.12 image and tested it without the value and it works fine.

Reply:
I see... I don't have a better solution. Approved